### DISCUSSION TITLE
Example of creating a mind map is needed

### DISCUSSION THREAD

**Discussion by devgeni** [xyflow/xyflow#1447] [Q&A]
This library looks impressive.
And I was hoping to find an example or guide on how to create a mind map.
Maybe someone already has an example and could share it with me?

**Comment by albatahmet** [comment]
[https://codesandbox.io/search?query=react%20flow&page=1&configure%5BhitsPerPage%5D=12](url)
You can find good examples here :smile:

**Reply by devgeni** [reply]
Thanks for the idea. I'll try to find an example there.

**Reply by albatahmet** [reply]
You're welcome
